rotkehlchen/chain/ethereum/modules/makerdao/sai/decoder.py

```diff
@@ -0,0 +1,535 @@
+                event.asset == self.sai and
+                event.balance.amount == amount_paid
+            ):
+                return None, []
+
+        if self.base_tools.is_tracked(depositor) is True:
+            event = HistoryBaseEntry(
+                event_identifier=transaction.tx_hash,
+                timestamp=ts_sec_to_ms(transaction.timestamp),
+                sequence_index=self.base_tools.get_sequence_index(tx_log),
+                location=Location.BLOCKCHAIN,
+                asset=self.sai,
+                event_type=HistoryEventType.SPEND,
+                event_subtype=HistoryEventSubType.PAYBACK_DEBT,
+                balance=Balance(amount=amount_paid),
+                location_label=depositor,
+                counterparty=CPT_SAI,
+                notes=f'Repay {amount_paid} {self.sai.symbol} to CDP {cdp_id}',
+            )
+            return event, []
+
+        return None, []
+
+    def _decode_liquidate_event(
+            self,
+            tx_log: EvmTxReceiptLog,
+            transaction: EvmTransaction,
+            decoded_events: list[HistoryBaseEntry],
+            all_logs: list[EvmTxReceiptLog],
+            action_items: Optional[list[ActionItem]],  # pylint: disable=unused-argument
+    ) -> tuple[Optional[HistoryBaseEntry], list[ActionItem]]:
+        """
+        This method decodes a liquidation event in a CDP
+
+        An example of such transaction is:
+        https://etherscan.io/tx/0x65d53653c584cde22e559cec4667a7278f75966360590b725d87055fb17552ba
+        """
+        liquidator = hex_or_bytes_to_address(tx_log.topics[1])
+        cdp_id = hex_or_bytes_to_int(tx_log.topics[2])
+
+        for event in decoded_events:
+            if (
+                event.event_type == HistoryEventType.SPEND and
+                event.event_subtype == HistoryEventSubType.SPEND and
+                event.asset == self.peth and
+                event.counterparty == CPT_SAI
+            ):
+                return None, []
+
+        # check for the transfer event of the liquidation to Maker SaiTap contract
+        for log in all_logs:
+            if len(log.topics) < 2:
+                continue
+
+            if hex_or_bytes_to_address(log.topics[1]) == MAKERDAO_SAITUB_CONTRACT and hex_or_bytes_to_address(log.topics[2]) == MAKERDAO_SAITAP_CONTRACT:  # noqa: E501
+                amount_raw = hex_or_bytes_to_int(log.data[:32])
+                amount = asset_normalized_value(amount=amount_raw, asset=self.weth)
+
+                event = HistoryBaseEntry(
+                    event_identifier=transaction.tx_hash,
+                    timestamp=ts_sec_to_ms(transaction.timestamp),
+                    sequence_index=self.base_tools.get_sequence_index(tx_log),
+                    location=Location.BLOCKCHAIN,
+                    asset=self.peth,
+                    balance=Balance(amount=amount),
+                    event_type=HistoryEventType.SPEND,
```

Comment:
I don't think SPEND + SPEND is what we use for liquidations or should use for liquidations. @yabirgb @nebolax did you not already start adding liquidations for other makerdao and aave? I would say liquidation needs a specific subtype for sure and should be shown in a different way in the UI

Reply:
Should I add a `HistoryEventSubType.LIQUIDATION`?

Reply:
Since atm subgraph is used for aave, for aave liquidations there is a dedicated structure `AaveLiquidationEvent`.

Reply:
So ... ehm ... nobody answered the question. @nebolax graph has nothing to do with decoders.

The questions is pretty clear. As part of your work as specced out by Yabir you need to decode liquidation events. What type/subtype did you guys use? Please communicate. You are a team.

Reply:
Dissapointing failure of communication. We resolved it in discord. The new type is added.